juju/juju
# feature: remove uvtool dependency

follow up to #6580

@perrito666 @babbageclunk 

This removes uvtool as a dependency, which should allow kvm containers to work on arm64, amd64, and ppc64el. QA has only been done by me on amd64 with vmaas.

**NB:** Since merging develop back into this branch there are some network updates that break kvm containers. When the rest of the network updates are complete and land it should be fine again. In order to test, once you have checked out the branch you will need to undo the changes so you can QA. Do that with `git revert -m 1 37a1a3b87fc54dc7391b4027e385ac48722ddf42 --no-commit`

QA:
1. `juju bootstrap vmaas21 kvm/purego --build-agent`
2. `juju add-machine`
3. `juju add-machine --series trusty`
4. `juju deploy ubuntu --to kvm:0`
5. `juju deploy mysql --to kvm:0`
6. `juju deploy wordpress --to kvm:1`
7. `juju deploy postgresql --to kvm:1 --series xenial`
8. `juju add-relation wordpress mysql`

verify that machines 0 and 1 came up with xenial and trusty respectively
verify that machines 0/kvm/0 cam up with xenial and 0/kvm/1 with trusty
verify that machine 1/kvm/0 came up with trusty and 1/kvm/1 with xenial
verify the relation was added...
why not create a wordpress blog to be sure. 

9. `juju remove-application ubuntu`
10. `juju remove-application postgresql`

verify that the disk images for those are gone ,by `juju ssh 0 (or 1)` and look in `/var/lib/juju/kvm/guests` to be sure they were removed.
